ecl/hql/hqldesc.cpp

```diff
@@ -136,35 +235,42 @@ void expandSymbolMeta(IPropertyTree * metaTree, IHqlExpression * expr)
         IHqlNamedAnnotation * symbol = queryNameAnnotation(expr);
         def->setProp("@name", str(expr->queryId()));
         def->setPropInt("@line", expr->getStartLine());
+        setFullNameProp(def, "@fullname", expr);
+
         if (expr->isExported())
             def->setPropBool("@exported", true);
         else if (isPublicSymbol(expr))
             def->setPropBool("@shared", true);
 
+        def->setProp("@inherit_type", getInheritTypeText(ihType));
+
         if (symbol)
         {
-            setNonZeroPropInt(def, "@start", symbol->getStartPos());
-            setNonZeroPropInt(def, "@body", symbol->getBodyPos());
-            setNonZeroPropInt(def, "@end", symbol->getEndPos());
+            def->setPropInt("@start", symbol->getStartPos());
```

Comment:
We are using start and body pos to extract definition signatures out of ecl files. Sometimes, a definition start at beginning of file which has start pos 0. Therefore, including start = 0 provides consistency in later processing and helps in debugging. 

But if it is a problem, I can revert this to original form and add conditionals in my ecldoc python code.